docs/articles/mirroring.md

```diff
@@ -0,0 +1,432 @@
+# OCI Registry Mirroring With zot
+
+> :point_right: A `zot` registry can mirror one or more upstream OCI registries.
```

Comment:
..., including popular cloud registries such as dockerhub and gcr.

@peusebiu pls. confirm

Reply:
Is this OK?  ", including popular cloud registries such as Docker Hub and Google Container Registry (gcr.io)."